vme/src/comm.cpp

```diff
@@ -211,7 +211,7 @@ void page_string(descriptor_data *d, const std::string &messg)
 
 void page_string(const descriptor_data *d, const std::string &messg)
 {
-    page_string(d, messg.c_str());
+    page_string(const_cast<descriptor_data *>(d), messg);
```

Comment:
Sorry for the noob questions. Why do you need to const_cast it when it is already a const?

Reply:
It works both ways - I'm casting away the const to call the other version.